src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public int getArea() {
+        return radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + this.getArea() + " radius: "
+                + this.radius + " units, color: " + this.getColor());
```

Comment:
```suggestion
        System.out.println("Figure: circle, area: " + getArea() + " radius: "
                + radius + " units, color: " + getColor());
```

do we need `this` keyword here?